ucp/_libs/core.pyx

```diff
@@ -445,8 +494,9 @@ class _Endpoint:
             raise UCXCloseError("send() - _Endpoint closed")
         nbytes = get_buffer_nbytes(buffer, check_min_size=nbytes,
                                    cuda_support=self._cuda_support)
+        tag = self._msg_tag_send
```

Comment:
Would it be alright to keep this as `self._msg_tag_send` below? It makes it a little easier for the reader to follow which tag is being referred to. Also should avoid having bugs accidentally creep in (due to a misunderstanding of what `tag` is).

Reply:
Good point